src/com/mostc/pftt/model/core/PhptSourceTestPack.java

```diff
@@ -142,6 +142,7 @@ public void cleanup(ConsoleManager cm) {
 		fs.deleteFileExtension(test_pack, ".skip.php");
 		fs.deleteFileExtension(test_pack, ".cmd");
 		fs.deleteFileExtension(test_pack, ".sh");
+		fs.deleteFileExtension(test_pack, ".clean.php");
 		fs.deleteFileExtension(test_pack, ".tmp");
```

Comment:
@lavturo but this change is not related to the `--CLEAN--` section, right? What you indeed touch is how `executeClean()` is called. That confused me a bit :)

Thanks.

Reply:
There is also a change I made where it creates the ".clean.php" (`PhptTestPreparer.java`), which is why this change is here. In case the test fails and it doesn't get deleted, this cleanup is supposed to delete the leftover files. 

`executeClean()` is located in `CliPhptTestCaseRunner.java `where it references the location of the ".clean.php" file to run. It currently works as is. 

Hope that clears the confusion.